lib/fathead/python_requests/parse.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin env python
+# -*- coding: utf-8 -*-
+from bs4 import BeautifulSoup
+from glob import glob
+import re
+from urllib.parse import urljoin
+
+
+def build_article(h2_parent, page_url):
+    '''Builds fathead article entry.
+
+    Accepts h2_parent and extracts title, url, abstract and code snippets.
+    Returns a list
+    '''
+    h2 = h2_parent.find('h2')
+    title = h2.text.replace('¶', '')
+    fragment = h2_parent.find('a').get('href')
+    url = urljoin(page_url, fragment)
+    abstract = ''
+    next_sibling = h2.find_next_sibling(text=None)
+    while next_sibling:
+        if next_sibling.name == 'p':
+            next_sibling_text = next_sibling.text.replace('\n', ' ')
+            abstract += ' {}'.format(next_sibling_text)
```

Comment:
We need to wrap this text, inside `<p>` tags, so we have 

```html
<section>
    <p> {{text}} </p>
    <pre><code> {{code}} </code></pre>
</section>
```

This will help to visually separate it from the code blocks. Right now the text is bunched up against the code block: https://beta.duckduckgo.com/?q=Transport+Adapters&ia=about&iax=1